src/components/pages/Documentation/Markdown/index.tsx

```diff
@@ -196,16 +197,21 @@ const Toggle: React.FC<{
           }
           onChange={(): void => updateToggleInd(toggleId, i)}
         >
-          {tab}
+          <div
+            className={cn('tab', styles.tab)}
+            style={{
+              minHeight: height
+            }}
+          >
+            {tab}
+          </div>
         </ToggleTab>
       ))}
     </div>
   )
 }
 
-const Tab: React.FC = ({ children }) => (
-  <div className={cn('tab', styles.tab)}>{children}</div>
-)
+const Tab: React.FC = ({ children }) => <>{children}</>
```

Comment:
If we're going to turn Tab into a dummy fragment component, I think we can just pass `React.Fragment` into `renderAst`'s components under `Tab`.

Reply:
Or now we wouldn't need to define the tab component on `renderAst`.  I think we don't have a use case where we use `tab` tag separately. So, the `tab` under `toggle` is parsed and rendered within the `Toggle` component itself.